indico/modules/events/registration/util.py

```diff
@@ -430,3 +430,27 @@ def build_registration_api_data(registration):
     registration_info['checkin_date'] = registration.checked_in_dt.isoformat() if registration.checked_in_dt else ''
     registration_info['event_id'] = registration.event_id
     return registration_info
+
+
+def generate_ticket_qr_code(registration):
+    """Generate a Pillow `Image` with a QR Code encoding a check-in ticket.
+
+    :param registration: corresponding `Registration` object
+    """
+    qr = QRCode(
+        version=6,
+        error_correction=constants.ERROR_CORRECT_M,
+        box_size=4,
+        border=1
+    )
+    config = Config.getInstance()
+    baseURL = config.getBaseSecureURL() if config.getBaseSecureURL() else config.getBaseURL()
+    qr_data = {"registrant_id": registration.id,
+               "checkin_secret": registration.ticket_uuid,
+               "event_id": unicode(registration.event_new.id),
+               "server_url": baseURL
+               }
```

Comment:
could we merge that into the previous line? i know it's technically allowed by PEP8 but we don't use this style anywhere else ;)